tests/stress/scopez_server/scopez_stress_test.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/env python3
```

Comment:
Can we rename this file to `scopez_server_stress.py` for consistency with waflz stress file?

Reply:
done